### DISCUSSION TITLE
Sensor support

### DISCUSSION THREAD

**Discussion by diegozalez** [FezVrasta/node-red-matter#18] [Ideas]
Hi first of all THANK YOU so much!!!! I've tried to integrate node-red into home kit and this is the best solution for me. The only thing I'm missing is support for more device types for example it will be really useful to have an environmental sensor, I'm willing to help but a bit lost on how to modify your code.

Again thanks for your awesome work!!!

**Comment by FezVrasta** [comment]
The device types have to be supported by the Matter.js library. I'm not sure what the status is right now. Once they are supported there, adding them to this library should be straightforward.

**Reply by Apollon77** [reply]
see https://github.com/project-chip/matter.js/tree/main/packages/matter.js#included-cluster-default-implementations ... most sensors have no command implementations and can so easiely be added
